src/blocks/remote-data-container/components/item-list/ItemList.tsx

```diff
@@ -157,7 +162,7 @@ export function ItemList( props: ItemListProps ) {
 			onChangeView={ onChangeView }
 			paginationInfo={ {
 				totalItems: totalItems ?? data.length,
-				totalPages: totalPages ?? 1,
+				totalPages: totalPages ?? ( hasNextPage ? page + 1 : page - 1 ),
```

Comment:
If neither `totalPages` nor `hasNextPage` are defined, I think we still want to default to `1`